blaserunning-and-shame/README.md

```diff
@@ -0,0 +1,56 @@
+**DRAFT**
+
+# Effects of the "Blaserunning" decree on shame
+
+[Society for Internet Blaseball Research](https://github.com/Society-for-Internet-Blaseball-Research)
+
+Authors: iliana etaoin (she/her)
+
+**Abstract**: We consider whether the "Blaserunning" decree in the ILB Season 3 election, which awards 0.1 runs for a team that successfully steals a base, impacts the number of games resulting in shame.
```

Comment:
remove "the" before "ILB"

Reply:
`the (ILB) election` or `ILB's election` seem right?

Reply:
stet. i'm wrong.